.dockerignore

```diff
@@ -0,0 +1,107 @@
+**/*.a
```

Comment:
is it possible to just symlink to .gitignore?

Reply:
Unfortunately they use [subtly incompatible formats](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/). Also they identify slightly different sets of files (I excluded some additional testing stuff that technically _is_ under revision control but it's not worth copying over).